asyncpg/pool.py

```diff
@@ -325,6 +404,9 @@ def create_pool(dsn=None, *,
     :param loop: An asyncio event loop instance.  If ``None``, the default
                  event loop will be used.
     :return: An instance of :class:`~asyncpg.pool.Pool`.
+
+    .. versionchanged:: 0.9.1
+       Pool prohibits use of its connections after they were released.
```

Comment:
An `InterfaceError` will be raised on any attempted operation on a released connection.